### PR TITLE
Fjabiyev/opt out c# sdk

### PR DESCRIPTION
**Pull request by FarhadJabiyev**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/communication/Azure.Communication.Sms/src/Models/OptOutChangeResponse.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.Communication.Sms.Models
+{
+    /// <summary> Response for an add and remove opt out request. Validate the returned items in the response to see which recipients were successfully added or removed from the opt outs list. </summary>
+    public partial class OptOutChangeResponse
```
Comment (FarhadJabiyev) [inline sdk/communication/Azure.Communication.Sms/src/Models/OptOutChangeResponse.cs:11]: Split it  for add/remove

File: sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs [hunk 2]
```diff
@@ -203,14 +204,191 @@ public async Task SendingSmsToNullNumberShouldThrow()
             Assert.Fail("SendAsync should have thrown an exception.");
         }
 
-        private void AssertHappyPath(SmsSendResult sendResult)
+        [Test]
+        public async Task CheckOptOutFromNullNumberShouldThrow()
```
Comment (FarhadJabiyev) [inline sdk/communication/Azure.Communication.Sms/tests/SmsClientLiveTests.cs:208]: Add more tests for validating incorrect number

**Review by phermanov-msft — APPROVED** [review]

**Review by gfeitosa-msft — APPROVED** [review]

**Comment by FarhadJabiyev** [comment]
@microsoft-github-policy-service rerun

**Comment by FarhadJabiyev** [comment]
@microsoft-github-policy-service agree

**Pull request merged**